pretext/Functions/GlobalVariables.ptx

```diff
@@ -108,8 +111,18 @@ z = square(x)
 print(z)
     </input>
   </program>
+
+
+  <note>
+
+  <p>
+    <alert>Instructor's Counsel:</alert> <em>Just because you can, doesn't mean you should.</em> Global variables exist,
+  but, as a novice programmer, you really should not use them. </p>
```

Comment:
Perhaps adding an exception is not a good idea, but I generally show students to define constants in the global scope.